### The next cell shows an example calculation of the difference in distance from my hometown (Houston, Tx) to the planet Jupiter versus the distance to Jupiter, California.

In [2]:
365000000-1901

364998099

### A summary of my distance difference findings

---

US towns used:
* Jupiter, California
* Jupiter, Florida
* Jupiter, North Carolina

Distance difference calculations (see example above):

|State of Jupiter| Miles from Houston| Miles Difference (vs planet Jupiter)|
|----------------|:-----------------------------:|:---------------------------------------------:|
|California|1901|364998099|
|Florida|1107|364998893|
|North Carolina|1004|364998996|